didcomm/did_doc/did_doc.py

```diff
@@ -0,0 +1,111 @@
+from abc import ABC, abstractmethod
+from typing import List
+
+from didcomm.common.types import JWK, KID
+
+
+class VerificationMethod(ABC):
+    """
+    DID DOC Verification method.
+    It can be used in such verification relationships as Authentication, KeyAgreement, etc.
+    """
+
+    @abstractmethod
+    def public_key_jwk(self) -> JWK:
```

Comment:
I think this and similar method name is confusing. Without deeply reading documentation i would expect it returns exactly `publicKeyJWK` property. Maybe it is better to rename it to `as_jwk`.